src/resolve-dependency.ts

```diff
@@ -176,6 +176,12 @@ function resolvePackage (name: string, parent: string, job: Job, cjsResolve: boo
     }
   }
 
+  if ('pnp' in process.versions) {
+    try {
+      return require.resolve(name, { paths: [parent] });
+    } catch (err) {}
+  }
+
```

Comment:
This is still PnP specific, this entire function should just use `require.resolve` to get the entry point (what is currently hardcoded to `<packageParent>/node_modules/name`) then continue its logic as is

Reply:
yes, but I don't know why the current logic doesn't do that, there could be a reason for the current resolve logic to be the way it is. I didn't want to break anything 😉